components/atoms/Banner.test.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+
+import { Primary } from "./Banner.stories";
+
+it("renders Banner in its primary state", () => {
```

Comment:
We want to do a11y checks with jest-axe as well. Take a look at the home.spec.js to see what that should look like, it is very simple. You will want to add a "describe" block as well to wrap the tests like in the home.spec.js

Reply:
Additionally you will want to put a test with jest-axe in components/atoms/Banner.test.js like this:

https://github.com/nickcolley/jest-axe#testing-react-with-react-testing-library